src/types/params.type.ts

```diff
@@ -26,7 +26,8 @@ export type InitParams = {
     | 'v14.0'
     | 'v15.0'
     | 'v16.0'
-    | 'v17.0';
+    | 'v17.0'
+    | 'v18.0;
```

Comment:
We now have v20.0 and soon meta is going to deprecate all version <= v18.0. Being said, can we please update this to the latest?